tests/routes.users.js

```diff
@@ -0,0 +1,96 @@
+const assert = require('assert');
+const httpStatus = require('http-status-codes');
+const supertest = require('supertest');
+const models = require('../models')
+const app = require('../app');
+const User = require('../models').User
+
+describe('GET /api/users', () => {
+    it('return all users', async () => {
+        const testUser = {
+            user: {
+                firstName: 'testName1',
+                age: '25'
+            }
+        };
+        await User.create(testUser.user);
+
+        const res = await supertest(app)
+            .get('/api/users')
+            .expect(httpStatus.OK);
+
+        if (res.body.user.id) {
+            assert(res.body.user, 'return correct user')
+        }
+    });
+});
+
+describe('POST /api/users', () => {
+    it('create one user', async () => {
+        const testUser = {
+            user: {
+                firstName: 'Name',
+                age: 30
+            }
+        };
+
+        const res = await supertest(app)
+            .post('/api/users')
+            .send(testUser)
+            .expect(httpStatus.OK);
+
+        const userById = await User.findByPk(res.body.user.id);
+
+        assert.deepStrictEqual(testUser.user.firstName, userById.firstName, 'create correct user');
```

Comment:
you need deep equal when you are comparing objects or arrays, here you simply compare strings, use strictEqual

Reply:
Also here you should check that the total users count in the database before + 1 = total users after the request